### PR TITLE
update checkpointer docs

### PR DESCRIPTION
**Pull request by felipemello1**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [ ] fix a bug
- [x] update tests and/or documentation
- [ ] other (please add here)

### CODE REVIEW

File: docs/source/deep_dives/checkpointer.rst [hunk 1]
```diff
@@ -58,39 +58,43 @@ from the `meta-llama website <https://llama.meta.com/llama-downloads>`_, you'll
     >>> for key, value in state_dict.items():
     >>>    print(f'{key}: {value.shape}')
 
-    tok_embeddings.weight: torch.Size([32000, 4096])
+    tok_embeddings.weight: torch.Size(128256, 3072])
```
Comment (acisseJZhong) [inline docs/source/deep_dives/checkpointer.rst:61]: nit: missing [

File: docs/source/deep_dives/checkpointer.rst [hunk 2]
```diff
@@ -58,39 +58,43 @@ from the `meta-llama website <https://llama.meta.com/llama-downloads>`_, you'll
...
     ...
     ...
     >>> print(len(state_dict.keys()))
-    292
+    255
```
Comment (acisseJZhong) [inline docs/source/deep_dives/checkpointer.rst:65]: should this be 255 or 147?

File: docs/source/deep_dives/checkpointer.rst [hunk 3]
```diff
@@ -58,39 +58,43 @@ from the `meta-llama website <https://llama.meta.com/llama-downloads>`_, you'll
...
 
 **HF Format**
 
 This is the most popular format within the Hugging Face Model Hub and is
 the default format in every torchtune config. This is also the format you get when you download the
-llama2 model from the `Llama-2-7b-hf <https://huggingface.co/meta-llama/Llama-2-7b-hf>`_ repo.
+llama3.2 model from the `Llama-3.2-3B-Instruct <https://huggingface.co/meta-llama/Llama-3.2-3B-Instruct>`_ repo.
 
-The first big difference is that the state_dict is split across two ``.bin`` files. To correctly
+The first big difference is that the state_dict is split across two ``.safetensors`` files. To correctly
```
Comment (SalmanMohammadi) [inline docs/source/deep_dives/checkpointer.rst:78]: nit
```suggestion
The first big difference is that the ``state_dict`` is split across two ``.safetensors`` files. To correctly
```

File: docs/source/deep_dives/checkpointer.rst [hunk 4]
```diff
@@ -145,16 +149,16 @@ Model Hub and is the default format in every torchtune config.
 For this checkpointer to work correctly, we assume that ``checkpoint_dir`` contains the necessary checkpoint
 and json files. The easiest way to make sure everything works correctly is to use the following flow:
 
-- Download the model from the HF repo using tune download. By default, this will ignore the "safetensors"
-  files.
+- Download the model from the HF repo using tune download. By default, this will ignore the "pth"
```
Comment (joecummings) [inline docs/source/deep_dives/checkpointer.rst:152]: Remove "by default" b/c that's not actually true

File: docs/source/deep_dives/checkpointer.rst [hunk 5]
```diff
@@ -145,16 +149,16 @@ Model Hub and is the default format in every torchtune config.
...
     .. code-block:: bash
 
-        tune download meta-llama/Llama-2-7b-hf \
-        --output-dir <checkpoint_dir> \
-        --hf-token <hf-token>
+       tune download meta-llama/Llama-3.2-3B-Instruct \
+       --output-dir /tmp/Llama-3.2-3B-Instruct \
+       --ignore-patterns "original/consolidated.00.pth"
 
 - Use ``output_dir`` specified here as the ``checkpoint_dir`` argument for the checkpointer.
```
Comment (joecummings) [inline docs/source/deep_dives/checkpointer.rst:163]: Remove?

Comment (felipemello1) [reply docs/source/deep_dives/checkpointer.rst:163]: sorry, remove what?

**Pull request closed without merging**